### DISCUSSION TITLE
Can you use GitHub App with the GH auth provider instead of GitHub OAuth App?

### DISCUSSION THREAD

**Discussion by ValentaTomas** [supabase/supabase#14577] [Questions]
I want to use the OAuth flow with the GitHub App instead of GitHub OAuth App. I actually managed to get it partially working but there are still some errors.

Have you tried to use auth this way?

**Comment by ValentaTomas** [comment]
Hey, any ideas about this?

**Comment by cfortuner** [comment]
bumping this!
